public/docs/ts/latest/cli-quickstart.jade

```diff
@@ -0,0 +1,261 @@
+:marked
+  ## Serve & test
+  
+  Run `ng serve` to launch the server, watch our files, and rebuild the app whenever we make changes.
+  
+  Open a browser on `http://localhost:4200/`; the app greets us with a message:
+  
+figure.image-display
+  img(src='/resources/images/devguide/cli-quickstart/app-works.png' alt="Our app works!")
+  
+.alert.is-important
+  :marked
+    _Windows Developers_: perform these commands in a console window opened as an administrator.
+    The Angular-CLI build steps create symlinks which require administrator rights under Windows.
+
+:marked
+  Now we're ready to write some code... or are we? We certainly could start writing code right now, but our tests would break.
+  
+:marked
+  ### Tests?
+  
+  What tests? The Angular-CLI created tests for our initial project! 
+  They are configured to run on [Google Chrome](https://www.google.com/chrome/) so make sure it is installed before running them.
+  
+  Open a new terminal (or console) window and run the *end-to-end (e2e)* tests with the command: `ng e2e`.
+  Look for the following output after a few console messages:
+code-example(format="").
+  Spec started
+  
+    angular2-cli-quickstart App
+      ✓ should display message saying app works
+  
+  Executed 1 of 1 spec SUCCESS in 2 secs.
+  
+:marked
+  Looks all green and good. Always a good sign. 
+  
+  Keep the server running in the original window and run the *unit tests* with the command: `ng test --build=false --watch=false`. 
+  After more console output, we should see a message like this:
+  
+code-example(format="").
+  Chrome 49.0.2623 (Windows 10 0.0.0): Executed 2 of 2 SUCCESS (0.016 secs / 0.009 secs)
+  
+:marked
+  It's a good idea to run tests before and after making changes to confirm that our app behaves as we expect it to.
+
+  *At last* we are ready to write some code.
+      
+a(id="edit")
+.l-main-section
+:marked
+  ## Edit the app
+  
+  We open the `angular2-cli-quickstart` projecd folder in our favourite editor.
+  
+  The root component is named after the project. We created the project with the name `angular2-cli-quickstart`, 
+  so our root component is `Angular2CliQuickstartApp` and can be found in `src/app/angular2-cli-quickstart.component.ts`
+  
+  Open it and update the title:
+  
++makeExample('cli-quickstart/ts/src/app/angular2-cli-quickstart.component.ts', 'title', 'src/app/angular2-cli-quickstart.component.ts')(format=".")
+  
+:marked
+  The browser reloads automatically and we see the revised title. That's nice, but we can make look better.
+  
+  Open `src/app/angular2-cli-quickstart.component.css` and give the app some style 
+
++makeExample('cli-quickstart/ts/src/app/angular2-cli-quickstart.component.css', null, 'src/app/angular2-cli-quickstart.component.css')(format=".")
+  
+:marked
+  Looking good!
+  
+figure.image-display
+  img(src='/resources/images/devguide/cli-quickstart/my-first-app.png' alt="Output of QuickStart app")
+
+:marked
+  ### Re-test and fix
```

Comment:
I have an issue with this label, the way it looks in the final product, seems like a footer for the image instead of a new section title.